src/auth/UserRequest.cc

```diff
@@ -132,6 +132,18 @@ Auth::UserRequest::denyMessage(char const * const default_message) const
     return getDenyMessage();
 }
 
+bool Auth::UserRequest::authenticated() const
```

Comment:
```suggestion
bool
Auth::UserRequest::authenticated() const
```

Reply:
changed.
I am fairly sure I had run astyle, I'm not sure how it got overlooked

Reply:
> changed. I am fairly sure I had run astyle, I'm not sure how it got overlooked

AFAICT from the [old test](https://github.com/squid-cache/squid/actions/runs/7937022245/job/21673425428#step:4:59) of reviewed code (i.e. before this change request suggestion was applied), astyle just does not correct this formatting in this case. I have not investigated why.